src/it/resources/com/google/checkstyle/test/chapter3filestructure/rule332nolinewrap/InputNoLineWrapBad.java

```diff
@@ -1,11 +1,13 @@
 package com.google.checkstyle.test. //warn
               chapter3filestructure.rule332nolinewrap;
 
-import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
+import static java.math. //warn
+        BigInteger.ZERO;
 
-import javax.accessibility. //warn
-    AccessibleAttributeSequence;
+import com.puppycrawl.tools.checkstyle.checks.design.FinalClassCheck; //ok
 import java.util.concurrent.atomic.AtomicIntegerFieldUpdater; //ok
+import javax.accessibility. //warn
+        AccessibleAttributeSequence;
```

Comment:
please make a changes to this file as min as possible. You add two lines and all other are changed for now reason.